jetty-util/src/main/java/org/eclipse/jetty/util/thread/QueuedThreadPool.java

```diff
@@ -364,11 +366,53 @@ public int getIdleTimeout()
     public void setIdleTimeout(int idleTimeout)
     {
         _idleTimeout = idleTimeout;
+        if (_idleTimeoutDecay > 1) {
+            // if non-default idleTimeoutDecay is configured, we must recompute _shrinkInterval
+            _shrinkInterval = computeShrinkIntervalNanos(idleTimeout, _idleTimeoutDecay);
+        }
         ReservedThreadExecutor reserved = getBean(ReservedThreadExecutor.class);
         if (reserved != null)
             reserved.setIdleTimeout(idleTimeout, TimeUnit.MILLISECONDS);
     }
 
+    /**
+     * @return the number of idle threads that are allowed to expire
+     * per idleTimeout interval.
+     */
+    @ManagedAttribute("number of idle threads allowed to die per idleTimeout interval")
+    public float getIdleTimeoutDecay()
```

Comment:
https://github.com/magibney/jetty.project/commit/dfab46bb3fff412683fbb847e08c9da649fa4088#r100643940
> why is this a float when the setter is an int?

Reply:
an oversight from a previous iteration of the design! thanks for pointing this out.

Reply:
fixed in 4a27d499ccae4bab1058c5b1505c085503eff07e